cfme/common/vm_console.py

```diff
@@ -140,9 +140,11 @@ def send_keys(self, text):
         """Send text to the console."""
         self.switch_to_console()
         canvas = self.provider.get_remote_console_canvas()
-        canvas.send_keys(text)
-        canvas.send_keys(Keys.ENTER)
         logger.info("Sending following Keys to Console {}".format(text))
+        for character in text:
+            canvas.send_keys(character)
+            wait_for(lambda: False, delay=0.3, num_sec=0.3, silent_failure=True)
```

Comment:
`wait_for` makes sense only when there is a condition to check, otherwise time.sleep is a perfectly valid solution. We usually have something to check for. Here, it probably isn't since it is sending keys to a canvas, so you can revert to using time.sleep, with a comment why it is used here, since the condition in wait_for is usually self-explaining. time.sleep with a comment why the sleep is needed will be much more readable than wait for false.

I know @quarckster suggested using wait_for, we do that when we see time.sleep, but we then sometimes have people come back and tell us there is nothing specific to check, so we likely do allow the use then.